src/ngraph/frontend/onnx_import/utils/reduction.hpp

```diff
@@ -101,6 +102,38 @@ namespace ngraph
                     reshape::get_default_axis_vector(op_node->get_shape().size()),
                     Shape{output_shape});
             }
+
+            template <class T>
+            std::shared_ptr<ngraph::Node> make_ng_arg_reduction_op(const Node& node)
```

Comment:
Ngraph is naming this class of operations as `IndexReduction` - this operators inherits from this class. Consider renaming it to sth  _index_-like.